config/sites.yaml

```diff
@@ -1,14 +1,8 @@
-# These are the sites where images will be *pushed*. Because of how the
-# image puller tool works, the only sites for now are {CHI,KVM}@TACC
+# The site where images will be *pushed*. Because of how the
+# image puller tool works, the only supported site, for now, is CHI@TACC.
```

Comment:
Are other sites not able to pull from the tacc bucket anonymously?

Reply:
was a reading comprehension fail 😅

Reply:
The other sites can pull from it. This is just the push tooling.

Reply:
We can configure it for anywhere (eg most of my dev was CHI@UC), I just figured I'd make this config the production case.

Reply:
Ah I think I see what you are saying. The wording is a bit unclear re: pull tooling. I mostly read this tool as interacting with where the production images get pushed to (so where all sites will pull from - the only place to pull from is this chameleon supported bucket at tacc), not that the only site that can pull from here is TACC.